model/src/main/kotlin/ScannerRun.kt

```diff
@@ -72,6 +73,12 @@ data class ScannerRun(
     @JsonSerialize(converter = ScanResultSortedSetConverter::class)
     val scanResults: Set<ScanResult>,
 
+    /**
+     * The scanners for each scanned package.
```

Comment:
The scanners for a package can also be empty. 
So, it should contain also IDs for not scanned packages.
How about saying instead:

The names of the scanners which have been used to scan the package.

Reply:
This is not addressed yet.